pkg/router/template/certmanager.go

```diff
@@ -12,10 +12,25 @@ import (
 	routeapi "github.com/openshift/origin/pkg/route/apis/route"
 )
 
+// certificateFile represents a certificate file.
+type certificateFile struct {
+	CertDir  string
+	ID       string
+	Contents []byte
```

Comment:
It doesn't look like anything uses the `Contents` field.

Reply:
For now its not used. I did have a comment re: staging the certificate writes as well - which is when we would have used it. But I think that's moot based on the call this morning, so will remove the `Contents` field.